Makefile

```diff
@@ -99,14 +99,14 @@ _fast_hooks:
 	$(MAKE) fast_compile -C ./hooks
 
 ext_gpp_link:
-	$(call echo,align_data = $(align_data); > Env.ld)
-	$(call echo,align_rdata = $(align_rdata); >> Env.ld)
-	$(call echo,align_bss = $(align_bss); >> Env.ld)
-	$(call echo,align_idata = $(align_idata); >> Env.ld)
+	$(call echo,align_data = '$(align_data);' > Env.ld)
+	$(call echo,align_rdata = '$(align_rdata);' >> Env.ld)
+	$(call echo,align_bss = '$(align_bss);' >> Env.ld)
+	$(call echo,align_idata = '$(align_idata);' >> Env.ld)
```

Comment:
Because of this, spam appeared when compiling locally
ld:Env.ld:1: ignoring invalid character `'' in script

Do you have any ideas how to remove spam?

Reply:
I do not unfortunately. I do not see those warnings when I run locally. It might be something with the setup where yours automatically considered it a string while default needs to be explicitly stated.